docs/build/sdk/indexs.md

```diff
@@ -33,106 +33,69 @@ the minimum amount.
 ```typescript
 const sdk = await SDK.initialize(endpoint);
 
-const poolId = await sdk.models.createCpmmMarketAndDeployAssets(
+const res = await sdk.models.createCpmmMarketAndDeployAssets({
   signer,
   oracle,
-  marketPeriod,
-  advised ? "Advised" : "Permissionless",
+  period: marketPeriod,
   marketType,
   mdm,
-  amts,
-  baseAssetAmount,
-  wts,
-  kp,
+  amount,
+  weights: weights.split(`,`),
   metadata,
-  false
-);
+  callbackOrPaymentInfo: false
+});
 ```
 
-**Arguments** | Name | Type | Introduction | | ---- | ---- | ------------ | |
-signer | KeyringPairOrExtSigner | The actual signer provider to sign the
-transaction. | | oracle | string |The address that will be responsible for
-reporting the market. | | period | MarketPeriod |Start and end block numbers or
-unix timestamp of the market. | | creationType | string |"Permissionless" or
-"Advised", Advised as default | | marketType | string |"Categorical" or "Scalar"
-| | mdm | MarketDisputeMechanism |Dispute settlement can be authorized, court or
-simple_disputes | | keep | string[] |Specifies how many assets to keep. | |
-weights | string[] |List of relative denormalized weights of each asset price. |
-| baseAssetAmount | Amount for native currency liquidity | | amounts | string[]
-| List of amounts of each outcome asset that should be deployed.| | metadata |
-DecodedMarketMetadata |Market metadata | | paymentInfo | |"true" to get txn fee
-estimation otherwise "false" |
+**Object Arguments** 
+| Name                  | Type                   | Description                                                    |
+| --------------------- | ---------------------- | -------------------------------------------------------------- |
+| signer                | KeyringPairOrExtSigner | The actual signer provider to sign the transaction             |
+| oracle                | string                 | The address that will be responsible for reporting the market  |
+| period                | MarketPeriod           | Start and end block numbers or unix timestamp of the market    |
+| marketType            | MarketTypeOf           | `Categorical` or `Scalar`                                      |
+| mdm                   | MarketDisputeMechanism | Dispute settlement can be authorized, court or simple_disputes |
+| metadata              | DecodedMarketMetadata  | A hash pointer to the metadata of the market                   |
+| amount                | string                 | The amount of each token to add to the pool                    |
+| weights               | string[]               | List of relative denormalized weights of each asset            |
```

Comment:
Maybe this is a typescript thing, but shouldn't the `amount` be a number, and the `weights` a vector/list of numbers?

Reply:
Yep, it is a typescript thing. We actually send them as strings while transacting with `polkadot.js/api`
https://github.com/zeitgeistpm/tools/blob/3d9c3df25b15ed0d2884ecb703240be9cd857345/packages/sdk/src/models/index.ts#L154